chore(curriculum): add accessibility transcripts (#58596)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/25-front-end-development/lecture-importance-of-accessibility-and-good-html-structure/672a55eb7791559421ff0cd3.md

Watch the lecture video and answer the questions below.

# --transcript--

What are good ways to make audio and video content accessible?

> Multimedia, especially video, has become the go-to format for sharing information online.
> With this surge in content, there’s a growing need to ensure everyone can access and enjoy video and audio content online, irrespective of their abilities or environment.

Reviewer: ```suggestion
Multimedia, especially video, has become the go-to format for sharing information online.

With this surge in content, there’s a growing need to ensure everyone can access and enjoy video and audio content online, irrespective of their abilities or environment.
```

Making your audio and video content accessible isn’t just a nice-to-have, it’s essential for reaching a wider audience.

> Let's look at some inexpensive ways to make your video and audio content more accessible.
> A video is not just about visuals but also audio, so the first thing you should consider doing is adding captions or subtitles to your video content.
> Captions provide the text version of spoken words and important non-verbal sounds, like music or laughter, synchronized with the video. 

Reviewer: ```suggestion
Let's look at some inexpensive ways to make your video and audio content more accessible.

A video is not just about visuals but also audio, so the first thing you should consider doing is adding captions or subtitles to your video content.

Captions provide the text version of spoken words and important non-verbal sounds, like music or laughter, synchronized with the video. 
```

Subtitles on the other hand are essential for people who don't understand the language you're speaking. This helps not only people who are deaf or hard of hearing but also those watching videos in noisy or quiet environments.

To add captions or subtitles to your video or audio content, you can use the `track` element inside your video or audio element:

```html
<!-- Video -->
<video controls src="video.mp4">
  <track
      src="captions.vtt" 
      kind="captions"
      srclang="en"
      label="English" 
  />
</video>

<!-- Audio -->
<audio controls src="audio.mp3">
   <track
       src="captions.vtt"
       kind="captions"
       srclang="en"
       label="English"
   />
</audio>
```

> The `kind` attribute is used to tell the track element how it should be used. Valid values for the kind attribute include `captions`, `subtitles`, `chapters`, and `metadata`.
> The `srclang` attribute represents the language for the `track` content. The `label` attribute is a descriptive title for the text track that browsers use to identify and display it in the list of available text tracks.

Reviewer: ```suggestion
The `kind` attribute is used to tell the track element how it should be used. Valid values for the `kind` attribute include `captions`, `subtitles`, `chapters`, and `metadata`.

The `srclang` attribute represents the language for the `track` content. The `label` attribute is a descriptive title for the text track that browsers use to identify and display it in the list of available text tracks.
```

Watch the lecture video and answer the questions below.

# --transcript--

What are good ways to make audio and video content accessible?

Multimedia, especially video, has become the go-to format for sharing information online.

> With this surge in content, there’s a growing need to ensure everyone can access and enjoy video and audio content online, irrespective of their abilities or environment.

Reviewer 2: ```suggestion
With this surge in content, there's a growing need to ensure everyone can access and enjoy video and audio content online, irrespective of their abilities or environment.
```

> Making your audio and video content accessible isn’t just a nice-to-have, it’s essential for reaching a wider audience.
> 
> Let's look at some inexpensive ways to make your video and audio content more accessible.

Reviewer 2: ```suggestion
Making your audio and video content accessible isn't just a nice-to-have, it's essential for reaching a wider audience. Let's look at some inexpensive ways to make your video and audio content more accessible.
```

A video is not just about visuals but also audio, so the first thing you should consider doing is adding captions or subtitles to your video content.

Captions provide the text version of spoken words and important non-verbal sounds, like music or laughter, synchronized with the video. 

Subtitles on the other hand are essential for people who don't understand the language you're speaking. This helps not only people who are deaf or hard of hearing but also those watching videos in noisy or quiet environments.

> To add captions or subtitles to your video or audio content, you can use the `track` element inside your video or audio element:

Reviewer 2: ```suggestion
To add captions or subtitles to your video or audio content, you can use the `track` element inside your `video` or `audio` element:
```

<!-- Audio -->
<audio controls src="audio.mp3">
   <track
       src="captions.vtt"
       kind="captions"
       srclang="en"
       label="English"
   />
</audio>
```

The `kind` attribute is used to tell the track element how it should be used. Valid values for the `kind` attribute include `captions`, `subtitles`, `chapters`, and `metadata`.

The `srclang` attribute represents the language for the `track` content. The `label` attribute is a descriptive title for the text track that browsers use to identify and display it in the list of available text tracks.

Another important thing to consider is providing a transcript for your audio and video content. A transcript is a text version of all the spoken words in your audio or video. Unlike captions, transcripts don’t need to be synchronized with the media. Transcripts are useful for deaf people and those hard of hearing. They're also beneficial for people who prefer reading instead of watching or listening. Transcripts also make your content searchable, allowing users to quickly find specific parts of your audio or video. If you have a video or audio on a website, you can simply add the captions below the audio or video:

```html

> <audio controls>
>    <source src="audio.mp3" />
>    Your browser does not support the audio element.
> </audio>
> 
> <!-- Transcript -->
> <h3>Transcript</h3>
> <p>
>    [Speaker 1]: Welcome to the tutorial on making accessible content
> </p>
> <p>
>    [Speaker 2]: Today, we'll cover captions, transcripts, and more.
> </p>

Reviewer 2: ```suggestion
<audio controls>
  <source src="audio.mp3" />
  Your browser does not support the audio element.
</audio>

<!-- Transcript -->
<h3>Transcript</h3>
<p>
  [Speaker 1]: Welcome to the tutorial on making accessible content
</p>
<p>
  [Speaker 2]: Today, we'll cover captions, transcripts, and more.
</p>

```
